src/FSharp.Build/Microsoft.FSharp.NetSdk.props

```diff
@@ -73,6 +73,11 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <PackageReference Include="System.ValueTuple" Version="$(ValueTupleImplicitPackageVersion)" />
   </ItemGroup>
 
+  <PropertyGroup Condition=" '$(ManagePackageVersionsCentrally)' == 'true' ">
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <DisableImplicitLibraryPacksFolder>true</DisableImplicitLibraryPacksFolder>
```

Comment:
@baronfel , how do we feel about : ManagePackageVersionsCentrally and Tool specific implicit versions?
It seems to me that if you are managing package versions centrally then we should at least warn if DisableImplicitFSharpCoreReference != '' and  DisableImplicitFSharpCoreReference != 'true' and ManagePackageVersionsCentrally = true, rather than magically using the implicit version.

Reply:
yeah, I can agree with that. That's something that we just added to the Dotnet.ReproducibleBuilds package (thanks to contributors!) and we're working on getting that released.

Separately, https://github.com/dotnet/sdk/issues/24797 may be a useful work issue here.